pkgs/applications/networking/dnscontrol/default.nix

```diff
@@ -13,6 +13,8 @@ buildGoModule rec {
 
   vendorHash = "sha256-BE/UnJw5elHYmyB+quN89ZkrlMcTjaVN0T2+h8cpPS8=";
 
+  subPackages = [ "." ];
+
```

Comment:
Can you build all package and move them to separate output ? Or their are not useful ?

Reply:
before

```console
$ ls result/bin
build  dnscontrol  generate  validate
```

after

```console
$ ls result/bin
dnscontrol
```

I think `dnscontrol` is the only useful package here

Reply:
They seems to be test https://github.com/StackExchange/dnscontrol/blob/bede608637dcb5de056358e81e28c84f21c9a4ec/documentation/unittests.md#built-in-tests can you at least run it in checkPhase ?